Bottom: 43d129f4f037ed88d95102704747aaaaad730f56
Top:    8790758ab4074ab11ab2cfab32e8ae9e8a8b204a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-04 18:51:31 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 795b9fe..c4b314e 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
